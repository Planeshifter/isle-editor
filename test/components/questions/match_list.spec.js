// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import Session from '@isle-project/session';
import Provider from '@isle-project/components/internal/provider';
import MatchListQuestion from '@isle-project/components/questions/match-list/main.js';


// VARIABLES //

const session = new Session( {}, true );


// TESTS //

describe( '<MatchListQuestion />', function test() {
	it( 'renders a match list question', () => {
		const editor = renderer.create( <Provider session={session}>
			<MatchListQuestion />
		</Provider> );
		expect( editor ).toMatchSnapshot();
	});
});
