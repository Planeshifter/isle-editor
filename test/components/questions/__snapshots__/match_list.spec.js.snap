// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MatchListQuestion /> renders a match list question 1`] = `null`;
